extra_geom/detectors.py

```diff
@@ -1678,6 +1678,13 @@ def _tile_slice(cls, tileno):
         fs_slice = slice(tile_fs_offset, tile_fs_offset + cls.frag_fs_pixels)
         return ss_slice, fs_slice
 
+    def position_modules(self, data, out=None, threadpool=None):
+        if isinstance_no_import(data, 'xarray', 'DataArray'):
+            # we shift module indices by one as JUNGFRAU labels modules starting from 1..
+            # position_modules returns a numpy array so labels disapear anyway.
+            data['module'] = data['module'] - 1
```

Comment:
```suggestion
            data = data.copy(deep=False)
            data['module'] = data['module'] - 1
```

If I'm following [the docs](https://xarray.pydata.org/en/stable/generated/xarray.DataArray.copy.html) correctly, this should avoid modifying the object that's passed in, but without copying all the data inside it.

Reply:
Good point, thanks!